### PR TITLE
Modify BitMessageBar component features (#811)

### LINKED ISSUE #811: BitMessageBar missing features
There are some missing parameters like `role` that must be added to the component.
The name of `MesageBarStyle.cs` file is incorrect. 
Some HTML attributes of the component elements are missed.
Some CSS properties of the component should be modified according to the BlazorUI standard style format.
Also, test methods of the component should be modified after changes.

### PR DESCRIPTION
**Pull request by maryamhdr**
This closes #811

### CODE REVIEW

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs [hunk 1]
```diff
@@ -102,10 +113,32 @@ private void ToggleExpandSingleLine()
...
-            messageBarIcon = MessageBarIconName.HasValue() ? MessageBarIconName : IconMap[MessageBarType];
-            return base.OnParametersSetAsync();
+            switch (messageBarType)
+            {
+                case BitMessageBarType.Blocked:
+                case BitMessageBarType.Error:
+                case BitMessageBarType.SevereWarning:
+                    return "alert";
+            }
+            return "status";
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs:125]: please move this line into the default case of the above switch statement.

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs [hunk 2]
```diff
@@ -102,10 +113,32 @@ private void ToggleExpandSingleLine()
...
+        private string GetAnnouncementPriority()
+        {
+            switch (messageBarType)
+            {
+                case BitMessageBarType.Blocked:
+                case BitMessageBarType.Error:
+                case BitMessageBarType.SevereWarning:
+                    return "assertive";
+            }
+            return "polite";
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs:137]: please move this line into the default case of the above switch statement.

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs (hunk 1 above)
Comment (maryamhdr) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs]: Done

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs (hunk 2 above)
Comment (maryamhdr) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs]: Done

**Review by msynk — APPROVED** [review]

**Pull request merged**